src/styles/_article.scss

```diff
@@ -9,13 +9,21 @@
     h4,
     p {
       margin-top: 20px;
+      a {
+        color: black;
+        text-decoration: underline;
+      }
+      strong {
+        display: inline;
+        font-weight: bold;
+      }
     }
     section {
       align-items: left;
       margin-bottom: 40px;
       max-width: 1260px;
       img {
-        max-width: 100%;
+        width: 100%;
```

Comment:
Oh wait, so this is scoped to only affect articles right? If yes, then fine. Doing this in the common.scss would have deeper implications.